app/templates/components/job-controls.hbs

```diff
@@ -6,16 +6,20 @@
     {{#if error}}
       {{#bs-alert type="danger"}}<span class="error-message">{{error}}</span>{{/bs-alert}}
     {{/if}}
-    {{#if job.isNew}}
+    {{#if job.isAuthorized}}
```

Comment:
What happens if a job is cancelled or encounters an error? Does something return it to **authorized** state? I couldn't find anything in bespin-api that did that.